src/date.ts

```diff
@@ -8,15 +8,14 @@ interface CoerceOptions {
 interface ValidationOptions extends CommonValidationOptions<Date> {
   maxFuture?: Duration;
   maxPast?: Duration;
+  format?: string;
```

Comment:
Is this supposed to be here? Seems not be be used, but would be more at home in the convertOptions.

Reply:
no and you gave the idea to include it as a built-in convert option, thx